src/Makefile

```diff
@@ -72,6 +76,18 @@ $(EXE): snabb bin
 	@echo -n "BINARY    "
 	@ls -sh $@
 
+core/version.lua:
```

Comment:
Request: Add an `else` to provide some meaningful version ID when `../.git` is not available.

(Ideally leave open the possibility for adding more ways to automatically detect the version ID in the future e.g. detecting a nix derivation name.)